content/applications/sales/sales/send_quotations/different_addresses.rst

```diff
@@ -9,7 +9,7 @@ everyone will have the same delivery location as the invoicing location.
 Activate the feature
 ====================
 
-Go to :menuselection:`Sales --> Configuration --> Settings` and activate the **Customer Addresses**
+Go to :menuselection:`Invoicing --> Configuration --> Settings` and activate the **Customer Addresses**
```

Comment:
Go to :menuselection:`Accounting app --> Configuration --> Settings` and scroll down to the :guilabel:`Customer Invoices` section. Then, check the box next to :guilabel:`Customer Addresses`, and click :guilabel:`Save`.